### ISSUE TITLE
How to use uuid type?

### ISSUE THREAD

**Issue by kesavkolla** [balderdashy/sails-postgresql#71]
I would like to use postgresql column type uuid for my model.  I also want to use default value to it as gen_random_uuid() function.  How to setup this in model definition?

Thanks in advance

**Comment by Globegitter** [comment]
@sgress454 @CWyrtzen I think this issue should be re-opened again. What @Iwansbrough mentioned is certainly a workaround for a development environment, but for production it makes sense to actually have `UUID` support since it has more efficient indexing for PK and FKs.

**Comment by arun057** [comment]
I specify

```
id: {
  type: 'uuid',
  primaryKey: true
}
```

and use [sails-migrations](https://github.com/BlueHotDog/sails-migrations) to manage migrations manually and set the default for id to `uuid_generate_v4`.

Note: you will have to install the `uuid-ossp` extension for postgresql.

```
CREATE EXTENSION uuid-ossp;
```

**Comment by sailsbot** [comment]
Thanks for posting, @kesavkolla.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
